Library/ViewModels/RewardCardViewModel.swift

```diff
@@ -181,17 +181,35 @@ private func backingReward(fromProject project: Project) -> Reward? {
     .coalesceWith(.noReward)
 }
 
+private func rewardDescription(project: Project, reward: Reward) -> String {
```

Comment:
In order to not repeat reward I'd suggest we rename this to something like `localizedDescription(project: Project, reward: Reward)` so that it could be called like so `map(localizedDescription(project:reward:))`. What do you think?

Reply:
I was wondering whether this would be more readable, but as soon as I posted it it seemed harder to read 🤦‍♂ 

```swift
let isBacking = project.personalization.isBacking
  let isNoReward = reward.isNoReward
  let userHasBacked = userIsBacking(reward: reward, inProject: project)

  switch (isBacking, isNoReward, userHasBacked) {
  case (false, true, true):
    return Strings.Thanks_for_bringing_this_project_one_step_closer_to_becoming_a_reality()
  case (false, false, _):
    return Strings.Back_it_because_you_believe_in_it()
  case (true, true, _):
    return Strings.Back_it_because_you_believe_in_it()
  default:
    return reward.description
  }
```